tests/lib/prepare.sh

```diff
@@ -1035,15 +1038,15 @@ EOF
         # * pc (to aid in debugging by modifying the kernel command line)
         # * core20 (to avoid the automatic refresh issue)
         if [ "$IMAGE_CHANNEL" != "$BASE_CHANNEL" ]; then
-            unsquashfs -d core20-snap core20.snap
-            snap pack --filename=core20-repacked.snap core20-snap
-            rm -r core20-snap
-            mv core20-repacked.snap $IMAGE_HOME/core20.snap
+            unsquashfs -d "${BASE}-snap" "${BASE}.snap"
```

Comment:
please check the overlap with the nested tests, because there it is already updated the kernel repack